### PR TITLE
Translation corrections

### PR DESCRIPTION
**Pull request by Tymaeus**
Various translation corrections.

### CODE REVIEW

File: Update/ura_004.txt [hunk 1]
```diff
@@ -236,7 +236,7 @@ void main()
 //rそれでも…知り合ったばかりの頃は、待ち合わせた店で何も注文せずにお冷だけを飲んでいたことを考えれば、少しは遠慮のない関係に近づけたのかもしれない。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "それでも…知り合ったばかりの頃は、待ち合わせた店で何も注文せずにお冷だけを飲んでいたことを考えれば、少しは遠慮のない関係に近づけたのかもしれない。",
-			NULL, "Still... compared to when they first met, even if she didn't order anything other than a cold drink, it seemed like she was feeling a little less reserved than back then.", Line_Normal);
+			NULL, "Still... compared to when they first met, it seemed like she was feeling a little less reserved around her, even if she didn't order anything other than a cold drink.", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/ura_004.txt:239]:
```suggestion
			NULL, "Still... compared to when they first met, when she didn't order anything at their meetings and just drank water, it seemed like she was feeling a little less reserved.", Line_Normal);
```

File: Update/ura_004.txt [hunk 2]
```diff
@@ -582,7 +582,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>客</color>", NULL, "<color=#f5e6d3>Customer</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bb_cusa1001", 256, TRUE);
 	OutputLine(NULL, "『…そもそも赤いなんとかってやつに、ロクなやつがいたためしがない』",
-			NULL, "\"...Even from the start, the red lady showed no signs of struggle.\"", Line_Normal);
+			NULL, "\"...In the first place, I've never met a red-something-or-other who turned out to be any good.\"", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/ura_004.txt:585]:
```suggestion
			NULL, "\"...In the first place, I've never met someone dressed in red who turned out to be any good.\"", Line_Normal);
```

File: Update/ura_004.txt [hunk 3]
```diff
@@ -593,7 +593,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>客</color>", NULL, "<color=#f5e6d3>Customer</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bb_cusb1001", 256, TRUE);
 	OutputLine(NULL, "『ブラックホールに引き込まれる隕石とか小惑星とかって、きっとああいう風に消えていくんだろうな…』",
-			NULL, "\"Everything vanished, like meteors and asteroids getting sucked into a black hole...\"", Line_Normal);
+			NULL, "\"I'm sure that's what a black hole looks like when it's devouring asteroids and planets...\"", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/ura_004.txt:596]:
```suggestion
			NULL, "\"I'm sure that's what a black hole looks like as it's devouring asteroids and planets...\"", Line_Normal);
```

File: Update/ura_004.txt [hunk 4]
```diff
@@ -1460,7 +1460,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#9d7f69>巴</color>", NULL, "<color=#9d7f69>Tomoe</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 28, "ps3/s14/28/vt4bb_tomoe1106", 256, TRUE);
 	OutputLine(NULL, "（それが、この子の根っこをダメにしているのね…）",
-			NULL, "(So, she's unable to reach the root of the problem...)", Line_Normal);
+			NULL, "(So, that corrupted the core of her being, huh...)", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/ura_004.txt:1463]:
```suggestion
			NULL, "(So, that corrupted her down to her core, huh...)", Line_Normal);
```

File: Update/ura_004.txt [hunk 5]
```diff
@@ -1484,20 +1484,20 @@ void main()
...
 	OutputLine(NULL, "（…それで、いいわけがない）",
-			NULL, "(...And there's no reason for it.)", Line_Normal);
+			NULL, "(...I can't just leave it at that.)", Line_Normal);
 	ClearMessage();
 
 //r…わかっている。お節介かもしれないと、誰に言われるまでもなく理解はしている。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "…わかっている。お節介かもしれないと、誰に言われるまでもなく理解はしている。",
-			NULL, "...She already knew that. Even though it might be seen as meddlesome, she recognized that without needing to be told.", Line_Normal);
+			NULL, "...She didn't need anyone telling her to know that might be considered meddlesome.", Line_Normal);
```
Comment (DoctorDiablo) [inline Update/ura_004.txt:1500]:
```suggestion
			NULL, "...She didn't need anyone saying anything to know that might be considered meddlesome.", Line_Normal);
```

**Review by DoctorDiablo — APPROVED** [review]

File: Update/ura_004.txt [hunk 6]
```diff
@@ -582,7 +582,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>客</color>", NULL, "<color=#f5e6d3>Customer</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bb_cusa1001", 256, TRUE);
 	OutputLine(NULL, "『…そもそも赤いなんとかってやつに、ロクなやつがいたためしがない』",
-			NULL, "\"...In the first place, I've never met a red-something-or-other who turned out to be any good.\"", Line_Normal);
+			NULL, "\"...In the first place, I've never met someone dressed in red who turned out to be any good.\"", Line_Normal);
```
Comment (tellowkrinkle) [inline Update/ura_004.txt:585]: The red here was the color of the marker they labeled the image with

**Pull request merged**